README.md

```diff
@@ -120,6 +120,16 @@ client.time_off.requests(:employeeId: employee_id, start: Time.now)
 client.time_off.whos_out(Time.now, "2014-12-31")
 ```
 
+### Using a custom API
+This is headed to reeplace completely the default Bamboo URL used to send and request data in order to make it more flexible. Specially useful for testing purposes.
+
+```ruby
+client = Bamboozled.client(subdomain: "your_subdomain", api_key: "your_api_key", api_url: "https://my.new/bamboo/url/")
+
+```
+
+If you set the <b>api_url</b> param when create the client, the gem will <b>ignore the subdomain</b> and directly will request information to the url you have set. Notice the structure of the URL, maintain it 
```

Comment:
I'm not sure if I get this text right, but here's my suggestion:
```suggestion
If you set the <b>api_url</b> param when calling the client method, the gem will <b>ignore the subdomain</b> param and it will request information directly to the api_url you have set. Please notice and follow the structure of the API URL.
```


WDYT?

Reply:
I'll edit a couple of things

Reply:
There're still syntax issues here:

If you set the <b>api_url</b> param [when create](https://ell.stackexchange.com/questions/14329/usage-of-when-followed-by-verbing) the client, the gem will <b>ignore the subdomain</b> and [directly will request ](https://learnenglish.britishcouncil.org/grammar/english-grammar-reference/where-adverbials-go-sentence) information to the url you have set.

Maybe just adjusting `when create` to `when creating` and `directly will request information` to `it will request information directly` will do.

Reply:
Ok 👍